sdk/src/client/utils.rs

```diff
@@ -164,17 +154,17 @@ impl Client {
     }
 
     /// Generates a new mnemonic.
-    pub fn generate_mnemonic() -> Result<String> {
+    pub fn generate_mnemonic() -> Result<Mnemonic> {
         generate_mnemonic()
     }
 
     /// Returns a seed for a mnemonic.
-    pub fn mnemonic_to_seed(mnemonic: Zeroizing<String>) -> Result<Seed> {
+    pub fn mnemonic_to_seed(mnemonic: &MnemonicRef) -> Result<Seed> {
```

Comment:
should we also allow a `Mnemonic`?

Reply:
Sure

Reply:
Actually, we can't, because `MnemonicRef` itself does not impl that.

Reply:
Isn't it auto-implemented for `MnemonicRef`?

Reply:
Ah, you are correct